ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace ServerCore.Pages.Submissions
 {
-    [Authorize(Policy = "IsEventAdminOrAuthorOfPuzzle")]
+    //TODO: Not working [Authorize(Policy = "PlayerCanSeePuzzle")]
```

Comment:
This page is used by all three roles?

Reply:
AuthorIndex is the one that looks like it should be used for Authors / Admins, although I think Jeremy has a PR out that might change that.